emitter/package.json

```diff
@@ -1,18 +1,23 @@
 {
-  "name": "test2",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "name": "js-interview-event-emitter",
+  "version": "2.0.0",
+  "description": "Testim JS interview",
   "scripts": {
-    "test": "mocha test/part1.test.js || exit 0",
-    "test2": "mocha test/part2.test.js || exit 0",
-    "tests": "mocha test/part1.test.js || exit 0",
-    "tests2": "mocha test/part2.test.js || exit 0"
+    "test:emitter:part1": "vitest --watch ./test/part1.test.ts",
+    "test:emitter:part2": "vitest --watch ./test/part2.test.ts",
+    "test": "vitest --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/testimio/JS-interview.git"
   },
-  "keywords": [],
   "author": "",
   "license": "ISC",
-  "devDependencies": {
-    "mocha": "^3.4.2"
+  "bugs": {
+    "url": "https://github.com/testimio/JS-interview/issues"
+  },
+  "homepage": "https://github.com/testimio/JS-interview#readme",
+  "dependencies": {
+    "vitest": "^2.1.8"
```

Comment:
@atlowChemi @barspielberg WDYT?

Reply:
<img src="https://media4.giphy.com/media/13d2jHlSlxklVe/giphy.gif"/>

Reply:
@omril1 @RoySegall I think vitest v2 is old, you should use v3

Reply:
Also, why do you need a package for a test runner in 2025?

Reply:
I would just use node test runner, you can skip installing dependencies at all since vitest is the only dep